lib/fat_free_crm/fields.rb

```diff
@@ -24,9 +24,10 @@ def has_fields
 
     module SingletonMethods
       def field_groups
-        if ActiveRecord::Base.connection.data_source_exists? 'field_groups'
+        begin
```

Comment:
sorry i made a mistake in my suggestion. this begin is redundant too

i (think, based on rubocop) it should look this.

```ruby
# good
def preferred
  ala
  bala
rescue StandardError => e
  something
end
```